### PR TITLE
Implemented pagination

### PR DESCRIPTION
**Pull request by bensuwun**
**Endpoint**: */getProducts*

Process:
- Front-end queries specified endpoint with `cursor` in query parameters.
- Function returns (if valid) array of 8 products or less, depending on where the cursor is currently pointed and how many products are left

### CODE REVIEW

**Comment by AdrielAmoguis** [comment]
Hey Ben!

Finally, pagination. Great job! Please append some unit tests in the `getProducts.spec.js` test suite before merging to properly test the implementation. This is to ensure that the integrity of the implementation will hold up as we add more and more features. Let's ask @zobhhz to create test-cases for these in tonight's scrum meeting. Just implement basic tests, then we'll leave it to her to create more detailed test scripts.

Sincerely,
Adi

**Review by AdrielAmoguis — APPROVED** [review]
Yeet. Approved for me, feel free to push.

**Comment by bensuwun** [comment]
medj pwet sorry HAHAHA arigathanks

File: __tests__/getProducts.spec.js
Comment (AdrielAmoguis) [inline __tests__/getProducts.spec.js:79]: Sorry, not too familiar yet with Jest's syntax. Does `expect(response.data.data.length <= 8)` work the same as `expect(response.data.data.length).toEqual(8)`?

Comment (bensuwun) [reply __tests__/getProducts.spec.js:79]: ah nono, the prior's literally just a js expression, it's possible kasi that a page might not be fully populated depending on (1) where the cursor is and (2) the number of products left, hence the less than condition

tho Jest's `.toEqual()` function is equivalent to Java String's `.equals` function, it checks the value rather than the reference

Comment (AdrielAmoguis) [reply __tests__/getProducts.spec.js:79]: Ah, alright. So it does return a boolean value and I'm guessing that `expect()` is expecting a `true`.

**Pull request merged**
